tests/unix_pipe.rs

```diff
@@ -124,10 +126,10 @@ fn event_when_receiver_is_dropped() {
 }
 
 #[test]
+#[cfg_attr(target_os = "nto", ignore = "Read close events are detected by NTO")]
 fn from_child_process_io() {
     // `cat` simply echo everything that we write via standard in.
     let mut child = Command::new("cat")
-        .env_clear()
```

Comment:
Why it was removed ? -->  `env_clear` clears the PATH variable and the system was unable to find `cat`. Is this necessary for the test framework?

Reply:
I think it's fine.